frontend/src/search-result.css

```diff
@@ -0,0 +1,7 @@
+.search-result-dialog {
```

Comment:
Definitely. I will sanitize this after merging feature/monitoring-feedback and feature/monitoring-tasks on feature/monitoring.